src/Logging/Logging.Console/src/ConsoleLoggerFormat.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Logging.Console
+{
+    /// <summary>
+    /// Format of ConsoleLogger messages.
+    /// </summary>
+    public enum ConsoleLoggerFormat
+    {
+        /// <summary>
+        /// Default format.
+        /// </summary>
+        Default,
+        /// <summary>
+        /// systemd '&lt;pri&gt;message' format.
```

Comment:
I think we can just describe what it does and leave the specific format to other docs. If you wanted to put in a `<remarks>` section about it that would be fine.

```suggestion
        /// Produces messages in a format suitable for systemd services.
```